.eslintrc.js

```diff
@@ -17,6 +17,9 @@ module.exports = {
     "no-console": 0,
     "indent": [2, 2, { "SwitchCase": 1, "MemberExpression": "off" }],
   },
+  globals: {
```

Comment:
I think we should set up a shim instead of using the global, see [cedar's example](https://github.com/Esri/ember-cli-cedar/blob/6ca5ea60b8c71e367e669403ce24857ad2ebf8c6/vendor/shims/cedar.js) and the [related import](https://github.com/Esri/ember-cli-cedar/blob/6ca5ea60b8c71e367e669403ce24857ad2ebf8c6/index.js).

In this case, it's a little trickier b/c of the multiple packages. The easiest thing would be to just shim the monoglobal (`arcgisRest`), so then in the files we'd use `import { setItemAccess } from 'arcgisRest'`. If you get that working, then we could look into using multiple shims so that we could do `import { setItemAccess } from '@esri/arcgis-rest-sharing'`. I think the way you do that is just change this just copy the original vendor file and update this line: `define('arcgisRest', [], vendorModule);` to `define('@esri/arcgis-rest-sharing', [], vendorModule);`.

Reply:
BTW, alternatively I think that https://github.com/ef4/ember-auto-import could be a good fit for this addon and the other rest-js addons (torii-provider-arcgis and ember-arcigs-server-services) - that would us out of the broccoli 🌳 trimming business entirely! While it gives me the heebie-geebies that it uses webpack in ember, and automatically parses your imports, I think that scoped to an addon I could get comfortable with that. [I've opened a couple issues over there](https://github.com/ef4/ember-auto-import/issues/created_by/tomwayson) so I can better understand how it would work for us. Maybe you want to wait until there's a response before adding the shims?

Reply:
@tomwayson knows far more about this stuff than I do. I just found ember-auto-import and thought it looked interesting. I would definitely defer to tom's judgment on this issue.

Reply:
@dbouwman said he was into this, but ember-auto-import is still a bit of a black box to me, and I don't have time to demystify it (i.e. figure out if we can get tree shaking).

I think the biggest issue is that we'd have to update the other addons that use rest-js (ember-cli-cedar, torii-provider-arcgis) at this time too. I'm already in the middle of reworking how ember-cli-cedar pulls in it's amcharts dep https://github.com/Esri/ember-cli-cedar/pull/100 and I just don't want to mess w/ all that right now. If I thought we'd be able to use dynamic `import()` to lazy-load amCharts over then there would be a very compelling reason to make the leap into ember-auto-import. However, amCharts (v3 at least) is a hot mess o global that has it's own internal method of lazy-loading plugins and assets.

I propose that we don't use ember-auto-imports at this time, and open an issue either here and/or in ember-cli-cedar to revisit, especially if/when cedar moves to amcharts v4 (which I just learned has it's own ["ember plugin"](https://github.com/amcharts/amcharts4#contents)).